thrift/src/main/thrift/atoms/timeline.thrift

```diff
@@ -0,0 +1,19 @@
+namespace * contentatom.timeline
+namespace java com.gu.contentatom.thrift.atom.timeline
+#@namespace scala com.gu.contentatom.thrift.atom.timeline
+
+include "entity.thrift"
+include "../shared.thrift"
+
+struct Timeline {
+  1: optional string typeLabel
+  2: optional list<string> tags
+  3: required list<TimelineItem> events
+  4: optional Entity entity
```

Comment:
A list probably makes sense, but perhaps they should be moved into `TimeLineItem` instead. Each event could be linked to a place, a person, an organisation. What do you think?